### PR TITLE
Add CODEOWNERS

### PR DESCRIPTION
**Pull request by aerorahul**
# Description
This PR:
- adds a `CODEOWNERS` file to the repository.  global-workflow is a collaborative space where contributions come from a variety of sources. This file will ensure that new development gets reviewed by the appropriate SME.

# Type of change
- [X] New feature (adds functionality)

# Change characteristics
- Is this a breaking change (a change in existing functionality)? NO
- Does this change require a documentation update? NO
- Does this change require an update to any of the following submodules? NO
  - [ ] EMC verif-global 
  - [ ] GDAS 
  - [ ] GFS-utils 
  - [ ] GSI 
  - [ ] GSI-monitor 
  - [ ] GSI-utils 
  - [ ] UFS-utils 
  - [ ] UFS-weather-model 
  - [ ] wxflow 

# How has this been tested?
There is no code change associated with this PR

# Checklist
- [ ] Any dependent changes have been merged and published
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have documented my code, including function, input, and output descriptions
- [ ] My changes generate no new warnings
- [ ] New and existing tests pass with my changes
- [ ] This change is covered by an existing CI test or a new one has been added
- [ ] I have made corresponding changes to the system documentation if necessary

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,210 @@
+# global-workflow is a collaborative space where contributions come from a variety of sources
+# This file is ensure that new development gets reviewed by the appropriate SME
```
Comment (KateFriedman-NOAA) [inline .github/CODEOWNERS]:
```suggestion
# This file is to ensure that new development gets reviewed by the appropriate SME
```

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+# jobs
+jobs/JGDAS_AERO_ANALYSIS_GENERATE_BMATRIX @CoryMartin-NOAA
+jobs/JGDAS_ATMOS_ANALYSIS_DIAG @RussTreadon-NOAA @CoryMartin-NOAA
+jobs/JGDAS_ATMOS_CHGRES_FORENKF @RussTreadon-NOAA @CoryMartin-NOAA
+jobs/JGDAS_ATMOS_GEMPAK @GwenChen-NOAA
+jobs/JGDAS_ATMOS_GEMPAK_META_NCDC @GwenChen-NOAA
+jobs/JGDAS_ATMOS_VERFOZN @EdwardSafford-NOAA
+jobs/JGDAS_ATMOS_VERFRAD @EdwardSafford-NOAA
+jobs/JGDAS_ENKF_* @RussTreadon-NOAA @CoryMartin-NOAA @CatherineThomas-NOAA
+jobs/JGDAS_FIT2OBS
```
Comment (KateFriedman-NOAA) [inline .github/CODEOWNERS]: Jack Woolen for `jobs/JGDAS_FIT2OBS`?

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -0,0 +1,210 @@
...
+jobs/JGDAS_GLOBAL_OCEAN_ANALYSIS_ECEN @guillaumevernieres
+jobs/JGDAS_GLOBAL_OCEAN_ANALYSIS_VRFY @guillaumevernieres
+jobs/JGFS_ATMOS_AWIPS_20KM_1P0DEG @GwenChen-NOAA
+jobs/JGFS_ATMOS_CYCLONE_GENESIS
+jobs/JGFS_ATMOS_CYCLONE_TRACKER
```
Comment (KateFriedman-NOAA) [inline .github/CODEOWNERS]: Jiayi for the CYCLONE and FSU_GENESIS job scripts?

**Review by KateFriedman-NOAA — COMMENTED** [review]
Looks good, thanks for adding this @aerorahul ! Left a few comments.

File: .github/CODEOWNERS [hunk 4]
```diff
@@ -0,0 +1,210 @@
...
+jobs/JGFS_ATMOS_FSU_GENESIS
+jobs/JGFS_ATMOS_GEMPAK @GwenChen-NOAA
+jobs/JGFS_ATMOS_GEMPAK_META @GwenChen-NOAA
+jobs/JGFS_ATMOS_GEMPAK_NCDC_UPAPGIF @GwenChen-NOAA
+jobs/JGFS_ATMOS_GEMPAK_PGRB2_SPEC @GwenChen-NOAA
+jobs/JGFS_ATMOS_PGRB2_SPEC_NPOESS @WenMeng-NOAA
+jobs/JGFS_ATMOS_POSTSND @BoCui-NOAA
+jobs/JGFS_ATMOS_VERIFICATION
+jobs/JGLOBAL_AERO_ANALYSIS_* @CoryMartin-NOAA
+jobs/JGLOBAL_ARCHIVE
```
Comment (DavidHuber-NOAA) [inline .github/CODEOWNERS]:
```suggestion
jobs/JGLOBAL_ARCHIVE @DavidHuber-NOAA
```

File: .github/CODEOWNERS [hunk 5]
```diff
@@ -0,0 +1,210 @@
...
+parsing_ufs_configure.sh @WalterKolczynski-NOAA @aerorahul @junwang-noaa
+preamble.sh @WalterKolczynski-NOAA
+product_functions.sh @WalterKolczynski-NOAA @aerorahul
+radmon_*.sh @EdwardSafford-NOAA
+rstprod.sh @WalterKolczynski-NOAA @DavidHuber-NOAA
+run_mpmd.sh @WalterKolczynski-NOAA @aerorahul @DavidHuber-NOAA
+syndat_getjtbul.sh
+syndat_qctropcy.sh
+tropcy_relocate.sh
+tropcy_relocate_extrkr.sh
```
Comment (DavidHuber-NOAA) [inline .github/CODEOWNERS]: Jiayi?

File: .github/CODEOWNERS [hunk 6]
```diff
@@ -0,0 +1,210 @@
...
+wave_*.sh @JessicaMeixner-NOAA @sbanihash
+
+# ush/python
+ush/python/pygfs/jedi/__init__.py @aerorahul @DavidNew-NOAA
+ush/python/pygfs/jedi/jedi.py @DavidNew-NOAA
+ush/python/pygfs/task/__init__.py @aerorahul
+ush/python/pygfs/task/aero_analysis.py @DavidNew-NOAA @CoryMartin-NOAA
+ush/python/pygfs/task/aero_bmatrix.py @DavidNew-NOAA @CoryMartin-NOAA
+ush/python/pygfs/task/aero_emissions.py @bbakernoaa
+ush/python/pygfs/task/aero_prepobs.py
```
Comment (DavidHuber-NOAA) [inline .github/CODEOWNERS]: Cory, David New?

**Review by DavidHuber-NOAA — CHANGES_REQUESTED** [review]

File: .github/CODEOWNERS (hunk 2 above)
Comment (aerorahul) [reply .github/CODEOWNERS]:
```suggestion
jobs/JGDAS_FIT2OBS @jack-woollen 
```

File: .github/CODEOWNERS (hunk 3 above)
Comment (aerorahul) [reply .github/CODEOWNERS]:
```suggestion
jobs/JGFS_ATMOS_CYCLONE_GENESIS @JiayiPeng-NOAA 
jobs/JGFS_ATMOS_CYCLONE_TRACKER @JiayiPeng-NOAA 
```

File: .github/CODEOWNERS (hunk 5 above)
Comment (aerorahul) [reply .github/CODEOWNERS]:
```suggestion
syndat_getjtbul.sh @JiayiPeng-NOAA 
syndat_qctropcy.sh @JiayiPeng-NOAA 
tropcy_relocate.sh @JiayiPeng-NOAA 
tropcy_relocate_extrkr.sh @JiayiPeng-NOAA 
```

File: .github/CODEOWNERS (hunk 6 above)
Comment (aerorahul) [reply .github/CODEOWNERS]:
```suggestion
ush/python/pygfs/task/aero_prepobs.py @CoryMartin-NOAA 
```

File: .github/CODEOWNERS [hunk 7]
```diff
@@ -0,0 +1,210 @@
...
+ush/python/pygfs/task/oceanice_products.py @aerorahul @GwenChen-NOAA
+ush/python/pygfs/task/snow_analysis.py @jiaruidong2017
+ush/python/pygfs/task/snowens_analysis.py @jiaruidong2017
+ush/python/pygfs/task/stage_ic.py @KateFriedman-NOAA
+ush/python/pygfs/task/upp.py @aerorahul @WenMeng-NOAA
+ush/python/pygfs/ufswm/__init__.py @aerorahul
+ush/python/pygfs/ufswm/gfs.py @aerorahul
+ush/python/pygfs/ufswm/ufs.py @aerorahul
+ush/python/pygfs/utils/__init__.py @aerorahul
+ush/python/pygfs/utils/marine_da_utils.py @guillaumevernieres @AndrewEichmann-NOAA
```
Comment (aerorahul) [inline .github/CODEOWNERS]:
```suggestion
ush/python/pygfs/utils/marine_da_utils.py @guillaumevernieres @AndrewEichmann-NOAA

```

**Review by DavidHuber-NOAA — APPROVED** [review]

**Review by KateFriedman-NOAA — APPROVED** [review]
Looks good, thanks for putting this together @aerorahul !

**Review by WalterKolczynski-NOAA — APPROVED** [review]

**Pull request merged**